src/AutoRest.CSharp/Common/Input/InputTypes/Examples/Serialization/TypeSpecInputExampleValueConverter.cs

```diff
@@ -53,12 +53,14 @@ private InputExampleValue CreateObject(ref Utf8JsonReader reader, JsonSerializer
         private const string ModelKind = "model";
         private const string ArrayKind = "array";
         private const string DictionaryKind = "dict";
+        private const string UnknownKind = "any";
```

Comment:
FYI in the next version of TCGC, this is changed to `unknown` align the type name in typespec.